internal/contour/cluster.go

```diff
@@ -143,6 +153,10 @@ func edscluster(svc *v1.Service, portString string, config *v2.Cluster_EdsCluste
 		cluster.CircuitBreakers = &v2cluster.CircuitBreakers{Thresholds: []*v2cluster.CircuitBreakers_Thresholds{thresholds}}
 	}
 
+	if upstreamProtocol == upstreamProtocolHTTP2 {
```

Comment:
given we're probably going to add TLS or something for http1/1+tls, what about a switch here
```
switch upstreamProtocol {
case upstreamProtocolHTTP2:
    cluster.Http2ProtocolOptions = &core.Http2ProtocolOptions{}
}
```

Reply:
This is the first thing I thought but then I supposed you'll prefer that if there weren't more than one case a single if is more appropriate! haha

Will change now.

Reply:
Done.